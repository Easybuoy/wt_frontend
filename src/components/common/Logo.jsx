import React from "react";
import styled from "styled-components";

import LogoImage from "../../assets/logo.png";

const StyledLogo = styled.div`
  img {
    width: 150px;
  }
`;

const Logo = () => {
  return (
    <StyledLogo>
      <img src={LogoImage} alt="Logo" />
    </StyledLogo>
  );
};

export default Logo;
