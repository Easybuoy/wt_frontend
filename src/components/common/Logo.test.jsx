import React from "react";
import { shallow } from "enzyme";

import Logo from "./Logo";

describe("<Logo />", () => {
  let wrapper;

  beforeEach(() => {
    wrapper = shallow(<Logo />);
  });

  it("renders the Logo component correctly", () => {
    expect(wrapper).toMatchSnapshot();
  });
});
