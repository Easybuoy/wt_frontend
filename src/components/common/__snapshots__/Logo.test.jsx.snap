// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Logo /> renders the Logo component correctly 1`] = `
<styled.div>
  <img
    alt="Logo"
    src="logo.png"
  />
</styled.div>
`;
